packages/react-core/src/components/Accordion/examples/AccordionDefinitionList.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Accordion, AccordionItem, AccordionContent, AccordionToggle } from '@patternfly/react-core';
+
+export const AccordionDefinitionList = () => {
+  const [expanded, setExpanded] = React.useState('ex-toggle2');
+
+  const onToggle = (id: string) => {
```

Comment:
This might turn into a broader discussion, but we might want to consider optimizing `examples` as they're converted to typescript.

Using this as an example, we could consider memoizing this function and including the anonymous callback used on line 19 as well.

@nicolethoen has added this as a topic to discuss in our next retro.

Reply:
I'll save most of my thoughts for the discussion tomorrow, but overall I worry that would be jamming too many different things into one PR, unless I drop the functional conversion or extraction that I'm currently doing with these TS conversions.